test/test.py

```diff
@@ -943,6 +954,15 @@ def test_single_quotes_in_single_quotes(self):
         # "a'b" == 'a'\''b'
         self.assertEqual(self.sh.run_command("prog quote '1\t"), "1'1\r\n")
 
+    def test_wordbreak_chars(self):
+        self.assertEqual(self.sh.run_command('prog break a\tc'), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command('prog break a:b:\tc'), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command('prog break a:b\tc'), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command("prog break 'a\tc'"), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command("prog break 'a:b\tc\t"), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command('prog break "a\tc"'), 'a:b:c\r\n')
+        self.assertEqual(self.sh.run_command('prog break "a:b\tc\t'), 'a:b:c\r\n')
```

Comment:
Yes, the tests with two tabs will trigger completion twice. For example, the last test is simulating the user doing the following:

1. Type `prog break "a:b`
1. Press TAB, completer returns `['a:b:c', 'a:b:d']`, shell fills in `:`
1. Type `c`
1. Press TAB, completer returns `['a:b:c']`, shell fills in `" `
1. Press enter, submitting `prog break "a:b:c" `